src/download.rs

```diff
@@ -28,9 +28,10 @@ pub struct Error(error::Error);
 type Result<T> = std::result::Result<T, Error>;
 
 const GIBIBYTE: i64 = 1024 * 1024 * 1024;
-const SNAPSHOT_BLOCK_WORKERS: usize = 64;
-const SNAPSHOT_BLOCK_ATTEMPTS: u8 = 3;
+const SNAPSHOT_BLOCK_WORKERS: usize = 2000;
+const SNAPSHOT_BLOCK_ATTEMPTS: u8 = 5;
 const SHA256_ALGORITHM: &str = "SHA256";
+const DISABLE_SHA: bool = true;
```

Comment:
We probably don't want to turn off the SHA hash checking by default.  I could maybe see providing users a flag to turn this off, but a constant isn't really configurable, and isn't currently documented or particularly discoverable.